.github/workflows/gen-db.yml

```diff
@@ -9,7 +9,6 @@ on:
   workflow_dispatch:  # manual execution
   release:
     types: [published]
-    branches: [main]
```

Comment:
Curious why this change? I guess we sometimes want to release from non-main branches?